bin/git-forgit

```diff
@@ -127,12 +127,53 @@ _forgit_quote_files() {
     echo "${files[*]}"
 }
 
-_forgit_pager=${FORGIT_PAGER:-$(git config core.pager || echo 'cat')}
-_forgit_show_pager=${FORGIT_SHOW_PAGER:-$(git config pager.show || echo "$_forgit_pager")}
-_forgit_diff_pager=${FORGIT_DIFF_PAGER:-$(git config pager.diff || echo "$_forgit_pager")}
-_forgit_ignore_pager=${FORGIT_IGNORE_PAGER:-$(hash bat &>/dev/null && echo 'bat -l gitignore --color=always' || echo 'cat')}
-_forgit_blame_pager=${FORGIT_BLAME_PAGER:-$(git config pager.blame || echo "$_forgit_pager")}
-_forgit_enter_pager=${FORGIT_ENTER_PAGER:-"LESS='-r' less"}
+_forgit_page() {
+    local pager
+    pager=${FORGIT_PAGER:-$(git config core.pager)}
+    if [[ -n $pager ]]; then
+        eval "$pager"
+    else
+        cat
+    fi
+}
+
+_forgit_diff_page() {
+    local pager
+    pager=${FORGIT_DIFF_PAGER:-$(git config pager.diff)}
+    if [[ -n $pager ]]; then
+        eval "$pager"
+    else
+        _forgit_page
+    fi
+}
+
+_forgit_show_page() {
+    local pager
+    pager=${FORGIT_SHOW_PAGER:-$(git config pager.show)}
+    if [[ -n $pager ]]; then
+        eval "$pager"
+    else
+        _forgit_page
+    fi
+}
+
+_forgit_blame_page() {
+    local pager
+    pager=${FORGIT_BLAME_PAGER:-$(git config pager.blame)}
+    if [[ -n $pager ]]; then
+        eval "$pager"
+    else
+        _forgit_page
+    fi
+}
+
+_forgit_enter_page() {
+    if [[ -n $FORGIT_ENTER_PAGER ]]; then
+        eval "$FORGIT_ENTER_PAGER"
+    else
+        LESS='-r' less
+    fi
+}
```

Comment:
It seems like these changes don't actually reduce the risks posed by eval?

If my understanding is correct, the potential risk should come from the output of `git config ...`, not the fallback string constants, such as `cat` or `LESS='-r' less`, etc.

Reply:
They do not and they don't intend to do. This was an attempt to solve #318 without spreading the usage of `eval` throughout the code base while using as little deferred code as possible. Not having the fallbacks as deferred code mainly was an attempt to improve readability.